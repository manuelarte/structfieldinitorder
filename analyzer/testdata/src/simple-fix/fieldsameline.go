package main

import "time"

type PersonFieldsSameLine struct {
	Name, Surname string
	Birthdate     time.Time
}

func mainComplex() {
	_ = PersonFieldsSameLine{ // want `fields for struct "PersonFieldsSameLine" are not instantiated in order`
		Birthdate: time.Now(),
		Surname:   "Doe",
		Name:      "John",
	}
}
