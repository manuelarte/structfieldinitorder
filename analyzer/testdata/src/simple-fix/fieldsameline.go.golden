package main

import "time"

type PersonFieldsSameLine struct {
	Name, Surname string
	Birthdate     time.Time
}

func mainComplex() {
	_ = PersonFieldsSameLine{ // want `fields for struct "PersonFieldsSameLine" are not instantiated in order`
		Name:      "John",
		Surname:   "Doe",
		Birthdate: time.Now(),
	}
}
